Makefile

```diff
@@ -2,6 +2,10 @@ all: clean package
 
 .PHONY: test
 test:
+	@echo ----- Running docker container for vault tests-----
+	docker run --cap-add=IPC_LOCK --rm -p 8200:8200 -d -e \
+		'VAULT_LOCAL_CONFIG={"backend": {"file": {"path": "/vault/file"}}, "listener":{"tcp":{"address":"0.0.0.0:8200","tls_disable":"true"}}}'\
```

Comment:
I believe we don't need this. We can definitely mock all these values in the test case itself. If you have written this to test the call to the vault server during the reveal, it's not required. It's like we are testing the vault, not the feature.

Reply:
Agreed, this should be covered in python tests, we should not pollute the Makefile with specific cases.

Reply:
Can make a separate section for this specific test. But would making the mock tests for vault feature be just like hard-coding the results?